__tests__/unit/init.js

```diff
@@ -7,5 +7,5 @@ jest.mock('@react-native-async-storage/async-storage', () =>
 
 jest.mock('react-native-reanimated', () => require('react-native-reanimated/mock'));
 jest.mock('react-native-gesture-handler', () =>
-  require('react-native-gesture-handler/dist/src/__mocks__/RNGestureHandlerModule'),
+  require('react-native-gesture-handler/jestSetup'),
```

Comment:
I want to pull attention to this diff.

After pulling from `main` and running `npm install`, the path `react-native-gesture-handler/dist/src/__mocks__/RNGestureHandlerModule` no longer existed. 

Changing it to `react-native-gesture-handler/jestSetup` fixed the tests, but I'm not entirely sure the implications of this change.

Reply:
If you check the podfile.lock it looks like `pod update` or similar was called and a few unrelated deps were updated (namely `RNGestureHandler`), you can either revert the unrelated changes, or stick with this, as long as the tests pass.

This begins an ongoing battle with fighting to keep the tests running in RN...